lib/screens/app/wallet/transfer_form.dart

```diff
@@ -257,3 +222,147 @@ class _TransferFormState extends State<TransferForm> with SingleTickerProviderSt
+  const AmountField({Key key, this.onChanged}) : super(key: key);
+
+  final Function onChanged;
+  @override
+  _AmountFieldState createState() => _AmountFieldState();
+}
+
+class _AmountFieldState extends State<AmountField> {
+  final controller = TextEditingController(text: '');
+  String inputString = "";
+  double seedsValue = 0;
+  double fiatValue = 0;
+  InputMode inputMode = InputMode.seeds;
+
+  @override
+  Widget build(BuildContext context) {
+    String balance = BalanceNotifier.of(context).balance.quantity;
+
+    return Column(
+      children: [
+        Stack(alignment: Alignment.centerRight, children: [
+          TextFormField(
+            keyboardType:
+                TextInputType.numberWithOptions(signed: false, decimal: true),
+            controller: controller,
+            autofocus: true,
+            validator: (val) {
+              String error;
+              double availableBalance =
+                  double.tryParse(balance.replaceFirst(' SEEDS', ''));
+              double transferAmount = double.tryParse(val);
+
+              if (transferAmount == 0.0) {
+                error = "Transfer amount cannot be 0.".i18n;
+              } else if (transferAmount == null || availableBalance == null) {
+                error = "Transfer amount is not valid.".i18n;
+              } else if (transferAmount > availableBalance) {
+                error =
+                    "Transfer amount cannot be greater than availabe balance."
+                        .i18n;
+              }
+              return error;
+            },
+            onChanged: (value) {
+              setState(() {
+                this.inputString = value;
+              });
+              widget.onChanged(_getSeedsValue(value));
+            },
+            decoration: InputDecoration(
+              filled: true,
+              fillColor: Colors.white,
+              focusedErrorBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(13),
+                  borderSide: BorderSide(color: Colors.amberAccent)),
+              errorBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(13),
+                  borderSide: BorderSide(color: Colors.redAccent)),
+              enabledBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(13),
+                  borderSide: BorderSide(color: AppColors.borderGrey)),
+              focusedBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(13),
+                  borderSide: BorderSide(color: AppColors.borderGrey)),
+              contentPadding: EdgeInsets.only(left: 15, right: 15),
+              hintStyle: TextStyle(
+                color: Colors.grey,
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(right: 15),
+            child: OutlineButton(
+              // shape: OutlineInputBorder( // TODO: needs design
```

Comment:
needed?

Reply:
well it took me a minute to find this code, but didn't look right... 

anyway deleting it ;)